### DISCUSSION TITLE
Chapter:6 ScannerError: mapping values are not allowed here

### DISCUSSION THREAD

**Discussion by akhilparakka** [smartcontractkit/full-blockchain-solidity-course-py#1623] [Q&A] [answered]
Tried to skip testing for Rinkeby but this  Error pops Up.
![Screenshot from 2022-08-01 15-18-23](https://user-images.githubusercontent.com/77268333/182122918-e8c8ad66-a94a-474d-b412-153166147167.png)

**Comment by cromewar** [comment, marked as answer]
Hello @Akhil-franko 
The error says something is wrong on your `brownie-config.yaml` so please share that script here and look for:
- Any typo on the code.
- Any indentation error.

**Reply by akhilparakka** [reply]
I'm really sorry. Had a typo. Should have been more careful. Sorry for wasting your time.
Thank You.

**Reply by cromewar** [reply]
Don't worry about it, typos are always the stone in our shoe as developers, I'm glad your problem is solved now.
